flight/uavobjectlib/Makefile

```diff
@@ -66,8 +66,15 @@ LDFLAGS += -Wl,--fatal-warnings
 SRC += $(filter-out $(OPUAVSYNTHDIR)/uavobjectsinit.c,$(wildcard $(OPUAVSYNTHDIR)/*.c))
 NO_AUTO_UAVO = YES
 
+# Backup the environmental variable before entering firmware-common.mk
```

Comment:
Is this a problem with our understanding of environment variables?  No other makefiles will see the changed value-- environment variables are only passed to child processes and a child is unable to modify its parent's environment) You shouldn't save / restore this IMO.